ui/src/utils/index.tsx

```diff
@@ -107,3 +108,13 @@ export function mapToRepoSyncStateT(status: string): RepoSyncStateT {
       return SYNC_STATUS.empty
   }
 }
+
+/**
+ * Method to get sync queue status
+ * @param syncQueue Sync type to evaluate
+ * @returns sync queue status
+ */
+export const getStatus = (syncQueue: RepoSyncQueue): RepoSyncStateT => {
+  const status = syncQueue?.hasError ? 'FAILED' : syncQueue?.status
```

Comment:
```suggestion
  const status = syncQueue?.hasError ? 'ERROR' : syncQueue?.status
```

Let's use the phrase "error" instead of "failed" where we can - just for consistency

Reply:
I think we will need to update this in a couple of other places

<img width="396" alt="image" src="https://user-images.githubusercontent.com/57259/194175516-d9a6b100-3a45-44e5-ba58-f9570eaf336d.png">

<img width="396" alt="image" src="https://user-images.githubusercontent.com/57259/194175543-d803677e-40ee-437a-8440-a8cca58a2174.png">